misc/version_updater.py

```diff
@@ -0,0 +1,325 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPL-3.0 2016-present Scille SAS
+
+"""
+Helper that help changing the version of a tool across the repository.
+"""
+
+import enum
+import re
+import sys
+from argparse import ArgumentParser
+from dataclasses import dataclass
+from fileinput import FileInput
+from pathlib import Path
+from typing import Callable, Dict, List, Tuple, Union
+
+ReplacementPattern = Union[str, Callable[[str], str]]
+
+RawRegexes = List["Replace"]
+
+ROOT_DIR = (Path(__file__) / "../..").resolve()
+
+
+@dataclass
+class Replace:
```

Comment:
The class name `Replace` (verb) suggests an action. What about one of the following: `ReplaceRegex`, `VersionRegex`, `Replacement` or `VersionReplacement` ?

Reply:
`ReplaceRegex` seems a better fit

Reply:
done